pkg/apis/numaflow/v1alpha1/const.go

```diff
@@ -127,9 +127,9 @@ const (
 	DefaultReadBatchSize    = 500
 
 	// Auto scaling
-	DefaultLookbackSeconds          = 180 // Default lookback seconds for calculating avg rate and pending
```

Comment:
Why the value changes? How do we evaluate that?

Reply:
for higher throughput pipelines I am seeing better autoscaling with 120, if we use 180 it will take 3 mins to reflect which will cause high back pressure in the pipeline, anyway it's a configuration change users can update it.